CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🔄 Hash indexes previously only worked with data of type `string`. They now
```

Comment:
I would prefer to merge this entry with the one for #632, and add a second PR link to the end of that. I think in general we should strive to keep the changelog context free.